Documentation/op-guide/supported-platform.md

```diff
@@ -12,6 +12,7 @@ The following table lists etcd support status for common architectures and opera
 | arm64        | Linux            | Experimental | @glevand         |
 | arm          | Linux            | Unstable     |                  |
 | 386          | Linux            | Unstable     |                  |
+| ppc64le      | Linux            | Stable       | etcd maintainers |
```

Comment:
@mkumatag 

Can you put your github handler there? It relies on the Jenkins IBM provides.